specifyweb/frontend/js_src/lib/components/WbToolkit/WbLeafletMap.tsx

```diff
@@ -24,24 +26,36 @@ export function WbLeafletMap({
   readonly mappings: WbMapping | undefined;
 }): JSX.Element {
   const [showLeafletMap, openLeafletMap, closeLeafletMap] = useBooleanState();
```

Comment:
Remove this now? that is, show the map when localityPoints !== undefined. So, openLeafletMap can be handleOpen (the one you have). closeLeafletMap will be ()=>setLocalityPoints(undefined). Don't see a performance problem (compared to what you have -- you recompute localityPoints anyways)

Reply:
fixed